combined_browser_metrics/views/use_counters.view.lkml

```diff
@@ -2,38 +2,44 @@ include: "//looker-hub/combined_browser_metrics/views/fenix_and_firefox_use_coun
 
 view: +fenix_and_firefox_use_counters {
 
-  dimension: submission_date {
+  dimension: sub_date {
```

Comment:
I think `submission_date` is the more common naming convention here?
but not sure, and won't block it for that